java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -135,4 +136,79 @@ public interface ServerManagementClusterCommands {
      * }</pre>
      */
     CompletableFuture<String> configResetStat(Route route);
+
+    /**
+     * Reads the configuration parameters of a running Redis server.<br>
+     * The command will be sent to a random node.
+     *
+     * @see <a href="https://redis.io/commands/config-get/">redis.io</a> for details.
+     * @param parameters An <code>array</code> of configuration parameter names to retrieve values
+     *     for.
+     * @return A <code>map</code> of values corresponding to the configuration parameters.
+     * @example
+     *     <pre>{@code
+     * Map<String, String> configParams = client.configGet(new String[] {"timeout" , "maxmemory"}).get();
+     * assert configParams.get("timeout").equals("1000");
+     * assert configParams.get("maxmemory").equals("1GB");
+     * }</pre>
+     */
+    CompletableFuture<Map<String, String>> configGet(String[] parameters);
+
+    /**
+     * Reads the configuration parameters of a running Redis server.
+     *
+     * @see <a href="https://redis.io/commands/config-get/">redis.io</a> for details.
+     * @param parameters An <code>array</code> of configuration parameter names to retrieve values
+     *     for.
+     * @param route Routing configuration for the command. Client will route the command to the nodes
```

Comment:
@param route  Specifies the routing configuration for the command. The client will route the command to the defined nodes.

Reply:
Done! (Will have to fix all commands with routes later on.)